ui/pages/smart-transactions/smart-transaction-status-page/smart-transaction-status-page.tsx

```diff
@@ -472,34 +479,43 @@ export const SmartTransactionStatusPage = ({
         flexDirection={FlexDirection.Column}
         alignItems={AlignItems.center}
         justifyContent={JustifyContent.center}
-        paddingLeft={10}
-        paddingRight={10}
+        paddingLeft={4}
+        paddingRight={4}
+        width={BlockSize.Full}
         style={{ flexGrow: 1 }}
       >
         <Box
           display={Display.Flex}
           flexDirection={FlexDirection.Column}
           alignItems={AlignItems.center}
+          paddingLeft={6}
+          paddingRight={6}
         >
           <SmartTransactionsStatusIcon
             iconName={iconName}
             iconColor={iconColor}
           />
           <Title title={title} />
+          <Deadline
+            isSmartTransactionPending={isSmartTransactionPending}
+            stxDeadline={stxDeadline}
+            timeLeftForPendingStxInSec={timeLeftForPendingStxInSec}
+          />
+          <Description description={description} />
+          <PortfolioSmartTransactionStatusUrl
+            portfolioSmartTransactionStatusUrl={
+              portfolioSmartTransactionStatusUrl
+            }
+            isSmartTransactionPending={isSmartTransactionPending}
+            onCloseExtension={onCloseExtension}
+          />
         </Box>
-        <Deadline
-          isSmartTransactionPending={isSmartTransactionPending}
-          stxDeadline={stxDeadline}
-          timeLeftForPendingStxInSec={timeLeftForPendingStxInSec}
-        />
-        <Description description={description} />
-        <PortfolioSmartTransactionStatusUrl
-          portfolioSmartTransactionStatusUrl={
-            portfolioSmartTransactionStatusUrl
-          }
-          isSmartTransactionPending={isSmartTransactionPending}
-          onCloseExtension={onCloseExtension}
-        />
+        {canShowSimulationDetails && (
+          <SimulationDetails
```

Comment:
I'm worried this may duplicate all our metrics, as they're tied to the rendering since many of the metrics are UI specific.

Not an issue with this PR though so the confirmations system team can test afterwards and fix the component if so, perhaps an optional `skipMetrics` property for example.